mm2src/coins/utxo/rpc_clients.rs

```diff
@@ -1594,25 +1654,38 @@ async fn electrum_request_multi(
     }
     drop(connections);
     if futures.is_empty() {
-        return ERR!("All electrums are currently disconnected");
+        return Err(JsonRpcError::new(
+            client.coin_name(),
+            request.clone(),
+            JsonRpcErrorType::Transport("All electrums are currently disconnected".to_string()),
+        ));
     }
 
-    match request {
+    match &request {
         JsonRpcRequestEnum::Single(single) if single.method == "server.ping" => {
             // server.ping must be sent to all servers to keep all connections alive
             return select_ok(futures)
                 .map(|(result, _)| result)
-                .map_err(|e| ERRL!("{:?}", e))
+                .map_err(|_| {
+                    JsonRpcError::new(
+                        client.coin_name(),
+                        request,
+                        JsonRpcErrorType::Transport("All electrums are currently disconnected".to_string()),
```

Comment:
Why did you change the error message to `"All electrums are currently disconnected"` here? Also, don't you think that other error types other than transport can be returned here? Maybe https://github.com/KomodoPlatform/atomicDEX-API/blob/6673d39b3aceb59337a8825d7e50ef8659d46615/mm2src/coins/utxo/rpc_clients.rs#L2793-L2798 need to return `JsonRpcError` error too.

Reply:
sorry that was a mistake..